kong/plugins/log-serializers/influxdb.lua

```diff
@@ -0,0 +1,30 @@
+local _M = {}
```

Comment:
Since this serializer is plugin specific, can you move the code into the plugin directory?